File: benchmark.go
package main

import (
	"fmt"
	"time"
)

func main() {
	start := time.Now()
	sum := 0

	for i := 1; i < 1000000001; i++ {
		sum += i
	}

	fmt.Printf("Sum: %d\n", sum)
	fmt.Printf("Time taken: %v\n", time.Since(start))
}

File: benchmark.rs
use std::time::Instant;

fn main() {
    let start = Instant::now();
    let sum: i64 = { 1..=1000000000 }.sum();
    let duration = start.elapsed();

    println!("Sum: {}", sum);
    println!("Time taken: {:?}", duration);
}
